README.md

```diff
@@ -87,3 +87,7 @@ plugins {
     id "com.ekino.oss.gradle.plugin.quality" version "1.0.0"
 }
 ```
+
+### Requirements
+
+Gradle 5.3.1 and JDK 8 are required.
```

Comment:
Gradle `5.6`?

Reply:
Nope, it works well with Gradle 5.3.1 ;)

Reply:
ok

Reply:
@clemstoquart Shouldn't we run the CI against this version to make sure it's still compatible though ? There should be a way to have multiple builds against different versions.

Reply:
@Crow-EH This is exactly the purpose of `GradleVersionsCompatibilityTest`.

Reply:
Oh my bad. That's perfect :ok_hand:

Reply:
We should go all in a explicitely tell that those are the *minimum* versions, or even give the current range of tested versions.

Reply:
Maybe. Feel free to open an issue or a PR ;)

Reply:
Ok I'll do it in its dedicated PR.